front_end/html/student_problem.html

```diff
@@ -172,11 +129,10 @@ <h2 style="display: inline">Expected output:</h2>
             function show_past_submissions() {
                 submissions_div = document.getElementById("submissions");
 
-                $.get("/get_submissions/{{ course_basics["id"] }}/{{ assignment_basics["id"] }}/{{ problem_basics["id"] }}/{{ user_id }}",
+                $.get("/get_submissions/{{ course_basics["id"] }}/{{ assignment_basics["id"] }}/{{ problem_basics["id"] }}/{{ student_id }}",
```

Comment:
Please use user_id rather than student_id for consistency reasons. Then we don't have to remember which one we are using in different places of the code.

Reply:
In this situation, the user_id refers to the user accessing this page, but the student_id refers to the specific student associated with the scores we're viewing.